c/dynalloc.c

```diff
@@ -29,6 +29,7 @@
 #include "zowetypes.h"
 #include "alloc.h"
 #include "dynalloc.h"
+#include "logging.h"
```

Comment:
We're introducing a new dependency. Can we work out a different way of returning debug info? Perhaps return and reason codes.